Podfile

```diff
@@ -15,15 +15,15 @@ abstract_target 'LeadKitAdditions' do
 
     use_frameworks!
 
-    pod "LeadKit", '~> 0.6.0'
+    pod 'LeadKit', :git => 'https://github.com/TouchInstinct/LeadKit', :branch => 'feature/text_field_view_model'
```

Comment:
what about podspec?

Reply:
just waiting for LeadKit release. will fix it now